packages/flutter/lib/src/rendering/grid.dart

```diff
@@ -643,7 +643,7 @@ class RenderGrid extends RenderVirtualViewport<GridParentData> {
     size = constraints.constrain(gridSize);
 
     if (callback != null)
-      invokeLayoutCallback(callback);
+      invokeLayoutCallback/*<BoxConstraints>*/(callback);
```

Comment:
I thought we could use real generic functions now.  I know you haven't landed the uber conversions patch, but maybe we should start using the real syntax for new instances?